### PR TITLE
Fix cruft update

### PR DESCRIPTION
**Pull request by ahuang11**
I botched the previous cruft update; this PR removes the extraneous `.rej` files and fixes publish docs. Also uses the latest cruft template, adds covert photo, saving block auth, and adds tutorial on ECS.

image

### Checklist

- [ ] References any related issue by including "Closes #<Issue Number>" or "Closes <Issue URL>".
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect-aws/issues/new/choose) first.
- [ ] Includes tests or only affects documentation.
- [ ] Passes `pre-commit` checks.
  - Run `pre-commit install && pre-commit run --all` locally for formatting and linting.
- [ ] Includes screenshots of documentation updates.
  - Run `mkdocs serve` view documentation locally.
- [ ] Summarizes PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-aws/blob/main/CHANGELOG.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,33 +18,37 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-aws) to see addit
...
-A list of available blocks in `prefect-aws` and their setup instructions can be found [here](https://PrefectHQ.github.io/prefect-aws/#blocks-catalog).
-
-### AWS Authentication
-
-You will need to obtain AWS credentials in order to use these tasks. Refer to the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for authentication methods available.
+Congrats! You can now easily load the saved block, which holds your credentials:
+ 
+```python
+from prefect_aws import Block
+Block.load("BLOCK_NAME_PLACEHOLDER")
```
Comment (desertaxle) [inline README.md:50]:
```suggestion
AwsCredentials.load("BLOCK_NAME_PLACEHOLDER")
```

**Review by desertaxle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ahuang11) [inline README.md:50]:
```suggestion
from prefect_aws import AwsCredentials
AwsCredentials.load("BLOCK_NAME_PLACEHOLDER")
```

Comment (desertaxle) [inline README.md]:
```suggestion
from prefect_aws import AwsCredentials
AwsCredentials.load("BLOCK_NAME_PLACEHOLDER")
```

Comment (ahuang11) [reply README.md]: Nice catch! Fixed the import too in the latest commit.

**Review by desertaxle — APPROVED** [review]

**Pull request merged**